Configure.cmake

```diff
@@ -422,3 +450,8 @@ endif()
 if (COMPILER_SUPPORTS_WEAK_ALIASES)
   set(COMMON_TARGET_DEFINITIONS ${COMMON_TARGET_DEFINITIONS} ENABLE_ALIAS=1)
 endif()
+
+# When cross compiling for ppc64, this bug-workaround is needed
+if(CMAKE_CROSSCOMPILING AND CMAKE_SYSTEM_PROCESSOR MATCHES "^(powerpc|ppc)64")
+  set(COMMON_TARGET_DEFINITIONS ${COMMON_TARGET_DEFINITIONS} UNDEF_USE_EXTERN_INLINES=1)
```

Comment:
I have commented all the places where you use this define, asking to add a comment saying that the use of `UNDEF_USE_EXTERN_INLINES` is platform specific.

I was thinking, instead of adding the comments, how about making the variable names specific to PPC64, so there is not doubt it target specific? May I ask you to change it to `POWER64_UNDEF_USE_EXTERN_INLINES`? With this name change, I am happy for you to avoid adding the comments all around.